src/include/rfac_func.h

```
@@ -1,110 +1,103 @@
+
+size_t rfac_lines(const char *buffer);
+size_t rfac_clines(const char *buffer);
+size_t rfac_nclines(const char *buffer);
+
+const char *rfac_version(void);
+
+void rfac_help(FILE *output);
+void rfac_info(void);
+
+const char *rfactor_name(rfactor_type type);
+void rfactor_print(FILE *stream, rfactor_type type, real r_min, real rr,
```

Clang (14.0.6)

[warning] parameter name 'rr' is too short, expected at least 3 characters

(readability-identifier-length)